fastplotlib/graphics/line.py

```diff
@@ -81,12 +79,7 @@ def link(self, event: str, target: Graphic, feature: str, new_data: Any, indices
             raise ValueError("event not possible")
 
         if event in self.registered_callbacks.keys():
-            self.registered_callbacks[event].append(EventData(target=target, feature=feature, new_data=new_data))
+            self.registered_callbacks[event].append(EventData(target=target, feature=feature, new_data=new_data, indices=None))
```

Comment:
indices are obtained when the event happens, not here

Reply:
if you rename `event` to `event_type` here in `link()` I think it'll help become clearer